stdlib/public/core/String.swift

```diff
@@ -973,6 +973,23 @@ extension String {
     return _nativeUnicodeUppercaseString(self)
 #endif
   }
+  
+  public // @testable
```

Comment:
This initializer is just `public`, it is not `@testable`.  `@testable` marks things that are public just for the sake of testing.